### PR TITLE
EN-97: Assignments updates

### PR DESCRIPTION
**Pull request by FedericoA15**
# Description :pencil:

Added to the 'endReason' assignment model as a string

## Link to ticket :link:

https://entropyteam.atlassian.net/browse/EN-97

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? :microscope:

image

### CODE REVIEW

File: src/main/java/com/entropyteam/entropay/employees/models/Assignment.java [hunk 1]
```diff
@@ -150,5 +153,8 @@ public boolean isActive() {
     public void setActive(boolean active) {
         this.active = active;
     }
-
+    public String getEndReason() {return endReason;}
```
Comment (FacuOrellana) [inline src/main/java/com/entropyteam/entropay/employees/models/Assignment.java:156]: minor: fix format

**Review by FacuOrellana — APPROVED** [review]
nice! just minor changes

File: src/main/java/com/entropyteam/entropay/employees/models/Assignment.java (hunk 1 above)
Comment (FedericoA15) [reply src/main/java/com/entropyteam/entropay/employees/models/Assignment.java:156]: okey

File: src/main/resources/db/migration/V32.0__DDL_update_assignment_table_update_ends_reason copy.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE assignment
+    ADD end_reason varchar(255) NULL;
```
[the remark this answers is not in the document]
Comment (FedericoA15) [reply src/main/resources/db/migration/V32.0__DDL_update_assignment_table_update_ends_reason copy.sql]: ready fixed

**Review by agustinb4 — APPROVED** [review]
Looks good!

**Pull request merged**